apps/server/test/integration/fixtures/updateNewFile.json

```diff
@@ -30,7 +30,6 @@
 				"units": "days"
 			},
 			"restrictions": {
-				"script": ["validateWithMagic(check dependece on another field)"],
```

Comment:
I take it this is not necessary after the refactor?

Reply:
It was in fact breaking tests after I removed script restrictions.

Reply:
ah! that's great! good riddance